code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -115,6 +115,7 @@
 	say_mod = "says"
 	hair_color = "mutcolor"
 	hair_alpha = 150
+	changesource_flags &= ~MIRROR_MAGIC

```

Comment:
you cant do this lol

Reply:
yeah I know, tried to throw it together fast. I've pushed a fix that actually compiles